package.json

```diff
@@ -50,49 +50,50 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/register": "^7.0.0",
-    "babel-preset-shopify": "^17.0.1",
+    "@babel/core": "^7.4.3",
+    "@babel/register": "7.4.0",
+    "@babel/runtime-corejs2": "^7.4.3",
+    "babel-preset-shopify": "^18.0.0",
     "chai": "^4.2.0",
-    "eslint": "^5.15.1",
+    "eslint": "^5.16.0",
     "eslint-find-rules": "^3.3.1",
     "eslint-index": "^1.5.0",
     "eslint-plugin-self": "^1.2.0",
-    "graphql": "^14.1.1",
+    "graphql": "^14.2.1",
     "istanbul": "^0.4.5",
-    "mocha": "^6.0.2",
-    "prettier": "^1.16.4",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4",
-    "typescript": "^3.3.3333"
+    "mocha": "^6.1.3",
+    "prettier": "^1.17.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "typescript": "^3.4.3"
   },
   "peerDependencies": {
     "eslint": "<6 >5.0.0",
     "prettier": "^1.14.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^1.5.0",
-    "@typescript-eslint/parser": "^1.5.0",
+    "@typescript-eslint/eslint-plugin": "^1.6.0",
+    "@typescript-eslint/parser": "^1.6.0",
     "babel-eslint": "10.0.1",
     "common-tags": "^1.8.0",
     "eslint-config-prettier": "4.1.0",
     "eslint-import-resolver-typescript": "^1.1.1",
-    "eslint-module-utils": "2.3.0",
+    "eslint-module-utils": "2.4.0",
     "eslint-plugin-babel": "5.3.0",
     "eslint-plugin-eslint-comments": "3.1.1",
     "eslint-plugin-graphql": "3.0.3",
-    "eslint-plugin-import": "2.16.0",
-    "eslint-plugin-jest": "22.3.0",
+    "eslint-plugin-import": "2.17.2",
+    "eslint-plugin-jest": "22.4.1",
```

Comment:
This adds [`no-empty-title`](https://github.com/jest-community/eslint-plugin-jest/pull/238).  Should we have a setting for that?